src/file_util.cc

```diff
@@ -19,87 +41,14 @@ bool ReadFileToString(const char* fn, std::string* contents) {
   return !file.fail();
 }
 
-class StartupDataCppWriter {
- public:
-  StartupDataCppWriter(const char* name, const char* filename,
-                       const std::string& data)
-      : name_(name),
-        filename_(filename),
-        data_(data),
-        file_(filename_, std::ios::binary) {}
-
-  bool Write() {
-    if (file_.bad()) {
-      return false;
-    }
-    WritePrefix();
-    WriteData();
-    WriteSuffix();
-
-    file_.close();
-    // printf("Wrote %s %d %s \n", name_, data_.size(), filename_);
-    return !file_.bad();
-  }
-
- private:
-  void WritePrefix() {
-    file_ << "// Autogenerated snapshot file. Do not edit.\n\n";
-    file_ << "#include \"third_party/v8/include/v8.h\"\n\n";
-    file_ << "namespace deno { \n\n";
-  }
-
-  void WriteSuffix() {
-    char buffer[500];
-    snprintf(buffer, sizeof(buffer), "v8::StartupData* StartupBlob_%s() {\n",
-             name_);
-    file_ << buffer;
-    snprintf(buffer, sizeof(buffer), "  return &%s_blob;\n", name_);
-    file_ << buffer;
-    file_ << "}\n\n";
-    file_ << "}  // namespace deno\n\n";
-  }
-
-  void WriteBinaryContentsAsCArray() {
-    char buffer[5];
-    for (size_t i = 0; i < data_.size(); i++) {
-      if ((i & 0x1F) == 0x1F) file_ << "\n";
-      if (i > 0) file_ << ",";
-      snprintf(buffer, sizeof(buffer), "%u",
-               static_cast<unsigned char>(data_.at(i)));
-      file_ << buffer;
+std::string Basename(std::string const& filename) {
+  for (auto it = filename.rbegin(); it != filename.rend(); ++it) {
+    char ch = *it;
+    if (ch == '\\' || ch == '/') {
```

Comment:
A little pedantic -- '\\' is not a valid path separator on Unix. 
That said, using the same logic on any platform is obviously great for testing.

Reply:
Yeah. Is there an easy CPP symbol to branch on? I'm fine with it as is... it's tested so if an error was ever caused here it should be easy to debug.

Reply:
That'd be `_WIN32`. But leave it for now.